frontend/src/tests/pages/School/SchoolIndexPage.test.js

```diff
@@ -55,7 +55,7 @@ describe("SchoolIndexPage tests", () => {
             expect(screen.getByText(/Create School/)).toBeInTheDocument();
         });
         const button = screen.getByText(/Create School/);
-        expect(button).toHaveAttribute("href", "/schools/create"); //fixme could have bug because plural
+        expect(button).toHaveAttribute("href", "/admin/schools/create"); //fixme could have bug because plural
```

Comment:
since you touched this line of code, you now have the responsibility to resolve the comment:
```
//fixme could have bug because plural
```

There's either a bug or there isn't.  If there is, fix it. If there isn't, remove this comment.